### PR TITLE
build the documentation of official packages

### PR DESCRIPTION
**Pull request by ringabout**
- [x] remove db stuffs
https://github.com/nim-lang/db_connector
ref https://github.com/nim-lang/packages/pull/2419
- [x] remove punycode
https://github.com/nim-lang/packages/pull/2412
- [x] build documentation

The documentation links subject to updates.

image

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -109,6 +109,16 @@
...
 
+- Several Standard libraries are moved to nimble packages, use `nimble` to install them:
+  - `std/punycode` => `pkg/punycode`
+  - `std/asyncftpclient` => `pkg/asyncftpclient`
+  - `std/smtp` => `pkg/smtp`
+  - `std/db_common` => `db_connector/db_common`
+  - `std/db_sqlite` => `db_connector/db_sqlite`
+  - `std/db_mysql` => `db_connector/db_mysql`
+  - `std/db_postgres` => `db_connector/db_postgres`
+  - `std/db_odbc` => `db_connector/db_odbc`
```
Comment (ghost) [inline changelog.md:125]: I think this list is a bit unclear as it doesn't directly say what to _install_ with Nimble - `pkg/punycode` or just `punycode`?

Comment (ringabout) [reply changelog.md:125]: Thanks for your suggestions.

Comment (ringabout) [inline changelog.md]:
```suggestion
  - `std/punycode` => `punycode`
  - `std/asyncftpclient` => `asyncftpclient`
  - `std/smtp` => `smtp`
  - `std/db_common` => `db_connector/db_common`
  - `std/db_sqlite` => `db_connector/db_sqlite`
  - `std/db_mysql` => `db_connector/db_mysql`
  - `std/db_postgres` => `db_connector/db_postgres`
  - `std/db_odbc` => `db_connector/db_odbc`
```

**Comment by ringabout** [comment]
ping

**Pull request merged**